test/presenters/notices/setup/prepare-return-forms.presenter.test.js

```diff
@@ -89,64 +91,220 @@ describe('Notices - Setup - Prepare Return Forms Presenter', () => {
       })
     })
 
-    describe('the "pageTitle" property', () => {
-      describe('when the "returnsFrequency" is "day"', () => {
-        beforeEach(() => {
-          dueReturnLog.returnsFrequency = 'day'
-        })
+    describe('when the "returnsFrequency" is "day"', () => {
+      beforeEach(() => {
+        dueReturnLog.returnsFrequency = 'day'
+      })
 
+      describe('the "pageTitle" property', () => {
         it('should return the relevant title', () => {
           const result = PrepareReturnFormsPresenter.go(session, dueReturnLog, recipient)
 
           expect(result.pageTitle).to.equal('Water abstraction daily return')
         })
       })
+    })
 
-      describe('when the "returnsFrequency" is "month"', () => {
-        beforeEach(() => {
-          dueReturnLog.returnsFrequency = 'month'
-        })
+    describe('when the "returnsFrequency" is "month"', () => {
+      beforeEach(() => {
+        dueReturnLog.returnsFrequency = 'month'
+      })
 
+      describe('the "pageTitle" property', () => {
         it('should return the relevant title', () => {
           const result = PrepareReturnFormsPresenter.go(session, dueReturnLog, recipient)
 
           expect(result.pageTitle).to.equal('Water abstraction monthly return')
         })
       })
 
-      describe('when the "returnsFrequency" is "quarter"', () => {
-        beforeEach(() => {
-          dueReturnLog.returnsFrequency = 'quarter'
-        })
-
-        it('should return the relevant title', () => {
+      describe('the "meterReadings" property', () => {
+        it('should return meter readings', () => {
           const result = PrepareReturnFormsPresenter.go(session, dueReturnLog, recipient)
 
-          expect(result.pageTitle).to.equal('Water abstraction quarterly return')
+          expect(result.meterReadings).to.equal([
+            [
+              [
```

Comment:
Either adds comments to columns / rows.

Or destructure the result meterReadings into args - 
```
const [firstPage] = meterReadings

const [firstColumnRows, secondColumnsRows] = firstPage

```